Formula/logstalgia.rb

```diff
@@ -1,10 +1,9 @@
 class Logstalgia < Formula
   desc "Web server access log visualizer with retro style"
   homepage "https://logstalgia.io/"
-  url "https://github.com/acaudwell/Logstalgia/releases/download/logstalgia-1.1.3/logstalgia-1.1.3.tar.gz"
-  sha256 "82e6a33c3c305c1f1d32d7e115ba0b307bb191ed2a70368a3cd9138ced0a98d9"
-  license "GPL-3.0"
-  revision 1
+  url "https://github.com/acaudwell/Logstalgia/releases/download/logstalgia-1.1.4/logstalgia-1.1.4.tar.gz"
+  sha256 "c049eff405e924035222edb26bcc6c7b5f00a08926abdb7b467e2449242790a9"
+  license "GPL-3.0-only"
```

Comment:
https://github.com/acaudwell/Logstalgia/blob/master/src/main.cpp#L1-L16
```suggestion
  license "GPL-3.0-or-later"
```

Reply:
Well, when pasting https://github.com/acaudwell/Logstalgia/blob/master/COPYING into https://tools.spdx.org/app/check_license/, it gives `GPL-3.0-only`, so did they mixed things up?

Reply:
@chenrui333  Why not use the content of the `COPYING` file?